docs/src/pages/ui/primitives/checkboxfield/react.mdx

```diff
@@ -0,0 +1,234 @@
+import { CheckboxField } from '@aws-amplify/ui-react';
+import { ControlledCheckbox, Demo } from './demo';
+import { Example } from '@/components/Example';
+
+CheckboxField is used to mark an individual item as selected, or to select multiple items from a list of individual items.
+
+## Demo
+
+<Example>
+  <Demo />
+</Example>
+
+## Usage
+
+Import the CheckboxField primitive and styles.
+
+```jsx
+import { CheckboxField } from '@aws-amplify/ui-react';
+import '@aws-amplify/ui-react/styles.css';
+
+<CheckboxField name="subscribe" value="true">
+  subscribe
+</CheckboxField>;
+```
+
+<Example>
+  <CheckboxField name="subscribe" value="true">
+    {'subscribe'}
+  </CheckboxField>
+</Example>
+
+### Controlled component
+
+```jsx
+import { useState } from 'react';
+
+import { CheckboxField } from '@aws-amplify/ui-react';
+import '@aws-amplify/ui-react/styles.css';
+
+const [checked, setChecked] = useState(false);
+
+<CheckboxField
+  name="subscribe"
+  value="true"
+  checked={checked}
+  onChange={(e) => setChecked(e.target.checked)}
+>
+  subscribe
+</CheckboxField>;
+```
+
+<Example>
+  <ControlledCheckbox />
+</Example>
+
+### Sizes
+
+Use the `size` prop to change the SelectField size. Available options are `small`, `large`, and none (default).
```

Comment:
Add examples for states such as isDisabled, isReadonly, hasError, etc?

Reply:
I think I put `isDisabled` into `### Visual options`. For `isReadOnly`, I didn't mention it that because html `read-only` prop essentially means for `value`. A `read-only` checkbox can still be checked or unchecked, which may make users confused. I will add `hasError` examples, and looks like I will have to add error styles, likewise for `Radio`